core/utils/ethabi.go

```diff
@@ -185,7 +204,12 @@ func EVMTranscodeJSONWithFormat(value gjson.Result, format string) ([]byte, erro
 	switch format {
 	case FormatBytes:
 		return EVMTranscodeBytes(value)
-
+	case FormatPreformattedHexArguments:
+		if !HasHexPrefix(value.Str) {
+			return nil, fmt.Errorf("%s input must be 0x-prefixed, got %s",
```

Comment:
This should have a test case.

Reply:
[Is this inadequate?](https://github.com/smartcontractkit/chainlink/blob/basic-multiword-ethabi-old/core/adapters/eth_tx_test.go#L85)